### PR TITLE
test(e2e): Add e2e tests

### PR DESCRIPTION
**Pull request by phoebeschmidt**
* Adds `test/e2e/` directory with cypress project (modified version of [the-example-app-e2e-tests](https://github.com/contentful/the-example-app-e2e-tests).
* Adds circle ci config
* tiny modifications to index.html (meta tag and title)

### CODE REVIEW

**Comment by phoebeschmidt** [comment]
In order to reuse CircleCI config, I created this branch with the unit-tests branch as the base. Pls review/merge that one first.

File: test/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  node:
+    version: "8"
+```
+
+#### 2. Download and install the tests:
+
+```
+dependencies:
+  pre:
+    - git clone https://github.com/contentful/the-example-app-e2e-tests.git ./test/e2e
```
Comment (Khaledgarbaya) [inline test/e2e/README.md]: Do we still need this? since we are shipping the tests with the repo

File: test/e2e/dictionaries.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  languages: {
+    dotnet: '.NET',
```
Comment (Khaledgarbaya) [inline test/e2e/dictionaries.js]: This also can be remove

File: test/e2e/README.md (hunk 1 above)
Comment (phoebeschmidt) [reply test/e2e/README.md]: true, i'm not doing this in the config. I'll clean up the readme.

**Pull request merged**